packages/wrangler/src/config/validation.ts

```diff
@@ -3383,6 +3393,40 @@ const validateSecretsStoreSecretBinding: ValidatorFn = (
 	return isValid;
 };
 
+const validateHelloWorldBinding: ValidatorFn = (diagnostics, field, value) => {
+	if (typeof value !== "object" || value === null) {
+		diagnostics.errors.push(
+			`"unsafe_hello_world" bindings should be objects, but got ${JSON.stringify(value)}`
+		);
+		return false;
+	}
+	let isValid = true;
+	// Secrets store bindings must have a binding, store_id, and secret_name
```

Comment:
This comment seems out of date

Reply:
Good catch. Thanks! Updated in https://github.com/cloudflare/workers-sdk/pull/9475/commits/3ed498a6101ab617d92fc1ea231be0456aa5c8b3.